assets/thanos-querier/deployment.yaml

```diff
@@ -37,7 +59,7 @@ spec:
             - if [ -x "$(command -v curl)" ]; then curl http://localhost:9090/-/healthy;
               elif [ -x "$(command -v wget)" ]; then wget --quiet --tries=1 --spider
               http://localhost:9090/-/healthy; else exit 1; fi
-        name: thanos-querier
+        name: thanos-query
```

Comment:
Note this is used and should be changed here as well https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/manifests/manifests.go#L2605

Reply:
Good catch! Updated.